_config.yml

```diff
@@ -120,6 +120,9 @@ include:
   - _users
   - _faqs
   - _release
+  - _service-center
```

Comment:
It's better to use the "getstart" directory to host the files, as we could add more examples here..

Reply:
Next, I want to bring together the “user manual” and the “getting started”, so I want to divide the directory with the project name. Do you have any better suggestions?

Reply:
how about we setup the directory just like this "getStarted/service-center", "userManual/serivce-center"?  
In this way we can keep adding files according to the sub project name.

Reply:
Ok, thanks. I will do it with your suggestions.

Reply:
@WillemJiang Please review again, thanks!